cypress/integration/login.feature

```diff
@@ -6,3 +6,9 @@ Feature: Login using Cognito
         When the user navigates to the front page
         Then the user sees that they are logged in
 
+    Scenario: User connects Author
+        Given that the user logs in with Feide for the first time
+        When they click OK in the Connect Author dialog

```

Comment:
Er dette hentet fra NVA-scenarios? Uansett burde dette spesifiseres mer synes jeg. Det er jo mange ting man kan gjøre når man logger seg inn for første gang; enten koble til forfatter som finnes, eller opprette ny forfatter, eller ingenting - som igjen fører til at vi ikke kommer videre. Men det finnes vel ikke en OK-knapp?

Reply:
Ser at det er hentet fra NVA-scenarios ja. Ikkje bra. Vi må vel uansett være mer spesifikk i vår beskrivelse i testingen, for det er jo ikke så enkelt som dette 👆

Reply:
Det er ikke helt match mellom mvp.feature og den virkelige verden ref. det jeg sa på møtet.

Reply:
Mhm. sant.